src/woo_publications/publications/tests/test_documents_model.py

```diff
@@ -0,0 +1,72 @@
+import uuid
+
+from django.db import IntegrityError, transaction
+from django.test import TestCase
+
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.test.factories import ServiceFactory
+
+from ..models import Document
+from .factories import PublicationFactory
+
+
+class TestDocumentApi(TestCase):
+    def test_document_api_constraint(self):
+        assert not Document.objects.exists()
+        publication = PublicationFactory.create()
+        service = ServiceFactory(
+            api_root="https://example.com/",
+            api_type=APITypes.drc,
+            oas="https://example.com/api/v1/oas",
+            header_key="Authorization",
+            header_value="Token 0cbccf9e-f9cd-4f9c-9516-7481e79989df",
+            auth_type=AuthTypes.api_key,
+        )
```

Comment:
at this point the exact value of `api_root`, key, auth etc. don't matter, as long as it's a service of type DRC.